### ISSUE TITLE
Please add comments view

### ISSUE THREAD

**Issue by ghost** [OpenHunting/openhunt#4]
I want to see other people's comments. The discussion around a product is good to see.

**Comment by jacquescrocker** [comment]
i agree with this. we'll need some systems in place to prevent spam or flame wars, but i think commenting is probably high priority

if anyone else has opinions on this (pro or con), please let us know!

**Comment by mvaragnat** [comment]
pro ! (at the very least, add comments count)

**Comment by patcito** [comment]
I think it was a mistake not to launch with this, many people are having a bad first impression. Public discutions are one of the top feature of PH. Worrying about spam right now is a bit premature, there are plenty of tools to prevent spam when it arrives, [askimet](http://akismet.com/development/api/) comes to mind but there are others. My 2c.

**Comment by jacquescrocker** [comment]
@patcito thanks for the feedback. hopefully the lack of comments at launch wont kill the site, but i agree its important and probably should be prioritized

**Comment by nblackburn** [comment]
@jacquescrocker This could link to #9 and only allow people with X karma to post comments.

**Comment by rossgeesman** [comment]
+1 on adding public commenting. Although I do think there is a place for both private (exposed only to product owner) and public comments and it would be great if the commenter could choose when submitting.

**Comment by wrightj2** [comment]
And another for seeing other people's comments, site is a great idea but lacks this key feature imo.

**Comment by nblackburn** [comment]
Perhaps the option to comment (_public_) or sense feedback (_private_) to distinguish the two would be nice.

You could take feedback one step further and allow for webhooks to be triggered upon submitting of feedback so developers can implement it directly into a ticketing or issue tracking system.
